exercises/clock/metadata.toml

```diff
@@ -1,4 +1,3 @@
 title = "Clock"
 blurb = "Implement a clock that handles times without dates."
 source = "Pairing session with Erin Drummond"
-source_url = "https://twitter.com/ebdrummond"
```

Comment:
I could only find Erin Drummond on LinkedIn, but LinkedId does a lot of rate limiting and I suspect it's not going to pass the link checker. It responds with status 999 to curl attempts, something that I learned about today that's possible 🤷